Xamarin.Forms.Core/Shell/Shell.cs

```diff
@@ -484,24 +484,25 @@ internal Task CompleteDeferredNavigating(ShellNavigatingEventArgs deferredArgs)
 			return GoToAsync(deferredArgs.Target, deferredArgs.Animate, false, deferredArgs);
 		}
 
-		public Task GoToAsync(ShellNavigationState state)
+		public Task GoToAsync(ShellNavigationState state, ShellParameter parameters = null)
```

Comment:
Just adding optional parameters is a breaking change you have to add each overload manually

https://haacked.com/archive/2010/08/10/versioning-issues-with-optional-arguments.aspx/